lib/assets/javascripts/turbograft/turbolinks.coffee

```diff
@@ -71,14 +71,14 @@ class window.Turbolinks
 
     rememberReferer()
 
-    options.partialReplace ?= false
-    options.onlyKeys ?= []
-    options.onLoadFunction = ->
-      resetScrollPosition() unless options.onlyKeys.length
-      options.callback?()
-
     fetchReplacement url, options
 
+  isPartialReplace = (options, response) ->
```

Comment:
I think it's a `visit` responsibility. We just don't have a `visit` object yet ;)